gnmi_server/server.go

```diff
@@ -183,7 +183,7 @@ func (srv *Server) Stop() {
 		log.Errorf("Stop() failed: not initialized")
 		return
 	}
-	s.Stop()
+	s.GracefulStop()
```

Comment:
>GracefulStop [](http://example.com/codeflow?start=3&length=12)

Is it true that `docker stop` will also trigger a graceful stop? If yes, I think we need to treat all the scenarios differently, only gracefully restart if rotating a cert.

Reply:
Yes that makes sense, docker stop will initially send a SIGTERM, if server has not stopped within a grace period (10s), docker stop will then send a SIGKILL which will force stop telemetry process. In most cases with this change, we will wait the 10s for the SIGKILL to come. I will change logic that if sigterm/sigint/sigquit/sighup is received we will call Stop() and not GracefulStop() which will be used only when rotating certs.